lib/checking_account.rb

```diff
@@ -0,0 +1,78 @@
+require 'csv'
+require_relative '../lib/account'
+
+module Bank
+
+  class CheckingAccount < Bank::Account
+
+      attr_reader :balance, :id, :open_date, :interest, :min_balance
+      attr_accessor :checks
+
+      def initialize(id, balance, open_date, checks)
+        @min_balance = 10
```

Comment:
You really need to use `super` here!  It would help DRY-up your code.  Thats **why** we use inheritance.